tests/test_processor_remote_tiles_wellington/test_processor_remote_tiles_wellington.py

```diff
@@ -19,22 +19,30 @@
 from src.geofabrics import processor
 
 
-class ProcessorRemoteTilesTest(unittest.TestCase):
+class ProcessorRemoteTilesWellingtonTest(unittest.TestCase):
     """ A class to test the basic processor class Processor functionality for remote tiles by downloading files from
-    OpenTopography within a small region and then generating a DEM. All files are deleted after checking the DEM."""
+    OpenTopography within a small region and then generating a DEM. All files are deleted after checking the DEM.
 
+    Tests run include:
+        1. test_correct_dataset - Test that the expected dataset is downloaded from OpenTopography
+        2. test_correct_lidar_files_downloaded - Test the downloaded LIDAR files have the expected names
+        3. test_correct_lidar_file_size - Test the downloaded LIDAR files have the expected file sizes
+        4. test_result_dem_windows/linux - Check the generated DEM matches the benchmark DEM, where the
+            rigor of the test depends on the operating system (windows or Linux)
+    """
```

Comment:
Not a change you need to make but it might be worth creating future lists like this in a bullet point form so you don't have to manually change the numbering if you change the order of the methods

Reply:
Nice - will move to that!